server/src/infra/websocket.adapter.ts

```diff
@@ -0,0 +1,28 @@
+import { INestApplicationContext } from '@nestjs/common';
+import { IoAdapter } from '@nestjs/platform-socket.io';
+import { createAdapter } from '@socket.io/postgres-adapter';
+import { ServerOptions } from 'socket.io';
+import { DataSource } from 'typeorm';
+import { PostgresDriver } from 'typeorm/driver/postgres/PostgresDriver.js';
+
+const CREATE_ATTACHMENT_TABLE = `
+  CREATE TABLE IF NOT EXISTS socket_io_attachments (
+      id          bigserial UNIQUE,
+      created_at  timestamptz DEFAULT NOW(),
+      payload     bytea
+  );
+`;
```

Comment:
Should this just go in our migrations instead?

Reply:
Maybe although then it will prompt to delete it cause we don't have an entity. We could make an entity and add it to the migration, idk.

Reply:
I think you could add it to migrations with no entity. If it does try to drop it, it would still try doing it this way when we generate migrations.

Reply:
It actually doesn't. You can see all the checks passed. TypeORM only generates migrations for entities or knows about.

Reply:
That's what I mean, so if you put it in the migrations folder, it should also do nothing.

Reply:
So I can move it to one instead.